lint-tasks.yml

```diff
@@ -1,18 +1,22 @@
 version: "3"
 
 vars:
+  NODEJS_18_BIN_DIR: "{{.TASKFILE_DIR}}/build/nodejs_18/node/bin"
```

Comment:
This is very minor so feel free to ignore but I prefer hyphens since they're easier to type, lol.